my_site/blog.php

```diff
@@ -0,0 +1,142 @@
+<!-- 
+ The blog for lab 10
+-->
+<?php
+    $current_page = 'todo';
+    $page_title = "To-Do List";
+    require 'common/required.php';
+?>
+<?php
+// MODAL for blod actual page
+session_start();
+
+if (!logged_in()) redirect('login.php?next=blog.php');
+
+$blog_posts = [];
+$path = '/JSON/blog_posts.json';
```

Comment:
The only error is that you have an extra '/' at the beginning. It understands it as a raw path, starting at the source of your computer. If you remove it (`'JSON/blog_posts.json'`), it understands it as a relative path, starting at the current directory, and then it works.